R/stan_module.R

```diff
@@ -117,6 +131,29 @@ setMethod(
     }
 )
 
+#' model_prep
+#'
+#' Populates the model section of a StanModule
+#'
+#' @param x A StanModule object
+#' @export
+model_prep <- function(x) {
+    if (length(x@priors) > 0) {
+        x@priors <- as.list(
+            paste(names(x@priors), "~", x@priors)
+        )
+
+        x@model <- paste0(
+            paste0(x@priors, collapse = "\n"),
+            "\n",
+            x@model
+        )
```

Comment:
I modified slightly the model prep function. I transfer the `x@model <- paste0(..` (line 146) inside the if loop because I want to modify the model section only in the case that priors are provided. If the priors are not provided, there is no need to change the model slot.

Reply:
lgtm